fast64_internal/sm64/sm64_objects.py

```diff
@@ -1403,6 +1421,719 @@ def draw_act(self, obj, layout, value):
+            layout.label(text=f"This will also export to /actors/{toAlnum(self.obj_name_gfx)}(.c, .h, _geo.c)")
+        else:
+            self.draw_level_path(layout)
+        layout.label(text=f"Geo Layout name will be: {self.geo_name}")
+        layout.label(text=f"Model ID will be: {self.model_id_define}")
+
+    def draw_obj_name(self, layout):
+        split_1 = layout.split(factor=0.55)
+        split_2 = split_1.split(factor=0.45)
+        split_2.label(text="Object Name")
+        split_2.prop(self, "object_name", text="")
+        if bpy.context.active_object:
+            tmp_obj_name = bpy.context.active_object.name
+        else:
+            tmp_obj_name = ""
+        split_1.label(text=f"or {repr(tmp_obj_name)} if no name")
+
+    def draw_bhv_options(self, layout):
+        if self.export_all_selected:
+            return
+        box = layout.box()
+        prop_split(box, self, "preset_behavior_script", "Preset Behavior Script")
+        box.operator("bone.add_behavior_script", text="Add Behavior Cmd").option = len(self.behavior_script)
+        for index, bhv in enumerate(self.behavior_script):
+            bhv.draw(box, index)
+
+    def draw(self, layout):
+        # level exports
+        layout.operator("object.sm64_export_level", text="Export Level")
+        self.draw_level_path(layout.box())
+        layout.separator()
+        # object exports
+        col = layout.column()
+        if not self.export_col and not self.export_bhv and not self.export_gfx:
+            box = col.box()
+            box.label(text="You must enable at least one export type", icon="ERROR")
+            self.draw_generic_options(col)
+            return
+        else:
+            col.operator("object.sm64_export_combined_object", text="Export Object")
+            self.draw_generic_options(col)
+
+        # bhv export only, so enable bhv draw only
+        if not self.export_col and not self.export_gfx:
+            return self.draw_bhv_options(col)
+
+        # pathing for gfx/col exports
+        prop_split(col, self, "export_header_type", "Export Type")
+
+        if self.export_header_type == "Custom":
+            prop_split(col, self, "custom_export_path", "Custom Path")
+        # always draw for level operator
+        prop_split(col, self, "level_name", "Level")
+        if self.level_name == "custom":
+            prop_split(col, self, "custom_export_name", "Level Name")
+
+        elif self.export_header_type == "Actor":
+            prop_split(col, self, "group_name", "Group Name")
+            if self.group_name == "custom":
+                prop_split(col, self, "custom_export_name", "Group Name")
+
+        if self.export_col and not self.export_all_selected:
+            prop_split(col, self, "collision_object", "Collision Obj")
+        if self.export_gfx and not self.export_all_selected:
+            prop_split(col, self, "graphics_object", "Geo Layout Obj")
```

Comment:
Is there a reason to abbreviate these?
Also fast64 uses "Geolayout" instead of "Geo Layout" everywhere else